apps/account/tests/test_models.py

```
@@ -1,146 +1,162 @@
 
 from apps.account.models import UserFriendship
-from apps.core.tests.setup import BaseTestSetUp
 from apps.room.models import UserConnectionToRoom
+from apps.room.tests.factories import RoomFactory
+from apps.transaction.tests.factories import ParentTransactionFactory
 
 
-class UserModelTestCase(BaseTestSetUp):
-    def test_str_method(self):
-        self.assertEqual(self.user.__str__(), self.user.name)
+def _build_image_bytes(width=64, height=64):
```

Ruff (0.14.7)

Missing return type annotation for private function `_build_image_bytes`

(ANN202)